projects/example/src/styles.scss

```diff
@@ -1 +1,2 @@
 /* You can add global styles to this file, and also import other style files */
+@import '@a-ui/core/dist/assets/styles/quarks';
```

Comment:
Why are you doing this?

Reply:
This was needed to get the VAR stylings from Antwerp, no?

![image](https://github.com/digipolisantwerp/location-viewer_widget_angular/assets/46924202/84f35856-93a8-4e0e-9f65-ca99472affc5)


![image](https://github.com/digipolisantwerp/location-viewer_widget_angular/assets/46924202/f62c87c8-fb36-4e71-882d-461c4c6c8b9d)

Reply:
Technically, yes. But if you load the core branding you already get these properties for free. The main reason why you would include quarks, is to use the (older) SASS variables (e.g. $spacer).

Reply:
The core branding is already imported, but I still didn't get the 'var' styling, am I doing anything wrong?

![image](https://github.com/digipolisantwerp/location-viewer_widget_angular/assets/46924202/24934f78-800a-4fc8-ada2-865dcde6394b)

Reply:
Oh, I see, you are using a deprecated version of the core branding. Please use 6.5.0 instead of 5.0.1.

Reply:
Great Success!!